gci/models.py

```diff
@@ -1 +1,109 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Students(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    program_year = models.ForeignKey('ProgramYear', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class Organizations(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    logo = models.ImageField(upload_to='', blank=True)
+    url = models.URLField(blank=True)
+    summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+    def __str__(self):
+        return self.name
+
+
+class Tasks(models.Model):
+
+    task_status = (
+        (1, 'Draft'),
+        (2, 'Published'),
+    )
+
+    name = models.CharField(max_length=200)
+    description = models.TextField(max_length=500)
+    status = models.IntegerField(choices=task_status, default=1)
+    max_instances = models.IntegerField(default=1)
+    mentors = models.ManyToManyField('Mentors')
+    tags = models.ManyToManyField('Tags')
+    is_beginner = models.BooleanField(default=False)
+    categories = models.ManyToManyField('Categories')
+    time_to_complete_in_days = models.IntegerField(default=3)
+    external_url = models.URLField()
+    metadata = models.TextField()
+    last_modified = models.DateField()
+    org = models.ForeignKey(Organizations, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class TaskInstance(models.Model):
+
+    task_instance = (
+        (1, 'CLAIMED'),
+        (2, 'ABANDONED'),
+        (3, 'SUBMITTED'),
+        (4, 'NEEDS_WORK'),
+        (5, 'OUT_OF_TIME'),
+        (6, 'COMPLETED'),
+        (7, 'PENDING_PARENTAL_C'),
+        (8, 'ONSENT'),
```

Comment:
messed up stuff? `PENDING_PARENTAL_C`?

Reply:
What mean by messed up stuff? `PENDING_PARENTAL_C` it was mentioned in the spec. 
Should I remove it?